lib/round-tracker.js

```diff
@@ -141,8 +141,8 @@ export async function mapCurrentMeridianRoundToSparkRound ({
     // this edge case by telling Postgres to ignore conflicts (`ON CONFLICT DO NOTHING)`
     await pgClient.query(`
       INSERT INTO meridian_contract_versions
-      (contract_address, spark_round_offset, last_spark_round_number)
-      VALUES ($1, $2, $3)
+      (contract_address, spark_round_offset, last_spark_round_number, first_spark_round_number)
+      VALUES ($1, $2, $3, $3)
```

Comment:
This is the first SPARK round managed by this IE contract version, therefore first and last spark round numbers are the same.